### PR TITLE
Image Gallery and moved to Cypress test suite

### PR DESCRIPTION
**Pull request by dominiquecuevas**
1. Gallery page using Cloudinary widget with carousel view of images
- Note: any new images added to Cloudinary need to be tagged 'gallery' to be included

https://github.com/athelia/full-spectrum-frontend/assets/56096225/fe67e185-34fb-480c-8405-c0f69c3e668a

2. Switched to Cypress testing from jest testing to allow for end-to-end testing on gallery page

3. Moved previous test for Header component to Cypress

Screen Shot 2023-07-12 at 11 44 51 PM

Screen Shot 2023-07-12 at 11 45 23 PM

4. Updated README for testing step

Screen Shot 2023-07-13 at 12 24 15 AM

### CODE REVIEW

File: cypress/e2e/gallery.cy.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    cy.get('@fourthIndexThumbnail')
+      .click()
+
+    cy.get('@fourthIndexImage')
+      .should('be.visible')
+  })
+
+  it('should view images on navigation button clicks', () => {
+    cy.get('[data-test=nav-next]')
+      .click().click().click().click().click()
```
Comment (athelia) [inline cypress/e2e/gallery.cy.ts]: Nit: for the `.click()` x 5 happening twice, is it worthwhile to wrap in a function?

File: cypress/fixtures/example.json
Comment (athelia) [inline cypress/fixtures/example.json:1]: Do we need this file?

File: cypress/support/commands.ts
Comment (athelia) [inline cypress/support/commands.ts:1]: Also this one - necessary?

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,10 +1,16 @@
 import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
 import Layout from '@/components/Layout'
+import Script from "next/script"
 
-export default function App({ Component, pageProps }: AppProps) {
+export default function App({ Component, pageProps }: AppProps) {
```
Comment (athelia) [inline pages/_app.tsx]: Nit: extra space end of line?

**Review by athelia — DISMISSED** [review]
Lots of nits - I'll approve but feel free to re-request review if you want

**Review by dominiquecuevas — COMMENTED** [review]
fixed nits and errors

**Review by athelia — APPROVED** [review]

**Pull request merged**